scripts/stablex/utilities.js

```diff
@@ -35,34 +48,45 @@ const fetchTokenInfo = async function(
   return tokenObjects
 }
 
-const addTokens = async function(token_addresses, web3, artifacts) {
-  const accounts = await web3.eth.getAccounts()
-
-  const BatchExchange = artifacts.require("BatchExchange")
-  const instance = await BatchExchange.deployed()
-
-  const TokenOWL = artifacts.require("../node_modules/@gnosis.pm/owl-token/build/contracts/TokenOWL")
-  const owl = await TokenOWL.at(await instance.feeToken.call())
+const addTokens = async function({ tokenAddresses, account, batchExchange, owl }) {
```

Comment:
I think your approach is the cleaner one: handing down the contract instances: batchExchange instead of the artifacts and the web3 object.

It is just that we have structured most other functions differently in the past. But since your approach is the "cleaner" one, lets stick with yours and amend the others over time